index.html

```diff
@@ -106,7 +105,7 @@ <h2 id="MyRecentWorks">My Recent Works</h2>
             alt="yoga image"
             id="yoga-image"
           />
-          <div class="project">
+          <div class="project" id="multi-post">
```

Comment:
- [ ] @sahlminkok and @VaheStepanyan100 great work on implementing the popup window, however, you forgot to remove the already created HTML code for the projects list, this whole section should be created dynamically using the information stored in your projects array, kindly check this [example](https://github.com/nicupop729/Portfolio/pull/11/commits/aabed37cc2208f52d0bd081a4cae8cdc83c49636.) to get ideas on how you can implement this feature. This will make it easy to add projects in the future.